Dockerfile

```diff
@@ -1,12 +1,8 @@
 FROM jujusolutions/jujubox:latest
+MAINTAINER Charles Butler <charles.butler@canonical.com>
 
-VOLUME "/home/ubuntu/.juju"
-VOLUME "/home/ubuntu/trusty"
-VOLUME "/home/ubuntu/builds"
-VOLUME "/home/ubuntu/layers"
-VOLUME "/home/ubuntu/interfaces"
-ADD install-review-tools.sh /install-review-tools.sh
-RUN /install-review-tools.sh
-# Override jujubox run.sh
-ADD run.sh /run.sh
-CMD /run.sh
+ENV LAYER_PATH=/home/ubuntu/charms/layers
```

Comment:
@chuckbutler by building off jujubox, does the charmbox dockerfile automatically 'inherit' stuff?  If so, could you just `s/ubuntu/$JUJU_USER` since it's defined in jujubox's dockerfile?

https://github.com/juju-solutions/jujubox/pull/21/files#diff-3254677a7917c6c01f55212f86c57fbfR4

Forgive the ignorance if dockerfiles don't stack like this..